src/raredisease.ts

```diff
@@ -94,41 +107,78 @@ function graph(data: DSVParsedArray<Datum>, container: HTMLElement) {
     .attr("viewBox", [-width / 2, -height / 2, width, height])
     .attr("style", "max-width: 100%; height: auto;");
 
+  // from Jackson Levitt's Pie Chart
+  const center = svg.append("g").attr("transform", `translate(${0}, ${0})`);
+
+  center
+    .append("circle")
+    .attr("fill", "white")
+    .attr("r", radius * 0.275);
+
+  center
+    .append("text")
+    .attr("text-anchor", "middle")
+    .attr("transform", `translate(0, 10)`)
+    //.attr("font-size", "0.75em")
+    .attr("font-family", "system-ui")
+    .attr("fill", "#999999")
+    .attr("id", "centerText")
+    .html("disease name");
+
   svg
     .append("g")
     .selectAll()
     .data(createPie(data))
     .join("path")
     .attr("fill", (d) => chakraToColor(d.data.chakra))
-    .attr("d", diseaseArc)
+    .attr("d", createArc)
+    .on("mouseenter", handleMouseOver)
```

Comment:
Used to be `mousemove` (triggers on every move) but I think we only need to trigger this once on enter.